src/components/header/Avatar.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { SFC } from 'react';
+import styled from '../../styled-components';
+
+import Image from '../layout/Image';
+
+export enum SIZE { small, large }
+interface Props {
+    size: SIZE.small | SIZE.large;
+    src: string;
+}
+
+const setSize = (size) => {
+    switch (size) {
+        case SIZE.small:    return '2rem';
+        case SIZE.large:    return '4rem';
+        default:            return '3rem';
+    }
+};
+
+const AvatarMask = styled.figure`
+    position: relative;
+    display: inline-block;
+    overflow: hidden;
+    width: ${props => setSize(props.size)};
```

Comment:
Can you reduce this to a single calculation in the attrs of the styled component. Be aware: this might not be friendly to TypeScript. (https://www.styled-components.com/docs/basics#attaching-additional-props)